src/main/java/com/qunite/api/web/controller/AuthenticationController.java

```diff
@@ -29,8 +31,10 @@
 @RestController
 public class AuthenticationController {
   private final UserService userService;
-  private final AuthResponseMapper responseMapper;
   private final UserMapper userMapper;
+  private final AuthResponseMapper authResponseMapper;
+  @Value("${jwt.access-token-expiration-time}")
+  private Integer jwtTokenExpirationTime;
```

Comment:
Mb place `@Value("${jwt.access-token-expiration-time}")` in AuthenticationResponse and remove this?

Reply:
How it should work in non-component class?)

Reply:
I do not like the way you done this. I think you should not invoke methods like `authenticationResponse.setExpires(jwtTokenExpirationTime);` in controller after authenticationResponse has been mapped. So mb extend TokenPair class with int expires and set expires in service?